### DISCUSSION TITLE
Turn OFF Camel Casing

### DISCUSSION THREAD

**Discussion by brianrhody** [cefsharp/CefSharp#3875] [General]
I'm updating an old application and the methods called from javascript are all upper cased and the defaulting to camel case is resulting in my having to update a lot of javascript.

Is there any way to turn OFF the camel casing?

**Comment by amaitland** [comment]
- http://cefsharp.github.io/api/94.4.x/html/P_CefSharp_IJavascriptObjectRepository_NameConverter.htm
- http://cefsharp.github.io/api/94.4.x/html/N_CefSharp_JavascriptBinding.htm

Set the property to null for no conversation, use a built in converter or implement your own.

```c#
chromiumWebBrowser.JavascriptObjectRepository.NameConverter = null;
```
